### PR TITLE
pngspan: First pass of fixes from first API day

### PR DESCRIPTION
**Pull request by ezequielgarcia**
Signed-off-by: Ezequiel Garcia <ezequiel@vanguardiasur.com.ar>

### CODE REVIEW

File: source/includes/_api_pngspan.md [hunk 1]
```diff
@@ -11,26 +11,48 @@ content-location: resource actually rendered. absolute start/end ts and either t
...
+
+PNG images can be retrieved for supporting metric visualization. PNG types include
+
+  * annt
+	* namespace - limits annt select to only entries within ns list
+	* uuid - limits annt selection to only entries within uuid list
+  * etag
+  * event
   * purge
 	* fflags=LOST - filter for only purges the lost data (within retention window)
```
Comment (jtallieu) [inline source/includes/_api_pngspan.md]: I don't understand the sentence - might need to be restated

File: source/includes/_api_pngspan.md [hunk 2]
```diff
@@ -43,31 +65,36 @@ Hit detection (for rollover) can be done in a browser by rendering opaque colors
...
+
 Parameter          		| Data Type     | Description   | Is Required
 ---------          		| -----------   | -----------   | -----------
-**start_timestamp**		| string        | Start Timestamp in EEN format: [YYYYMMDDHHMMSS.NNN](#een-timestamp) | true
-**end_timestamp**  		| string        | End Timestamp in EEN format: [YYYYMMDDHHMMSS.NNN](#een-timestamp) | true
-**width**         		| int        	| Width in pixels of resulting PNG. Must be an integer greater than 0. | true
-**id**         			| string        | Camera Id | true
-**foreground_color**    | string        | Color of foreground (active). If both fg and bg have 0 for alpha, assumed fully opaque (0xff). 32bit ARGB color. | true
-**background_color**    | string        | Color of background (inactive). 32bit ARGB color. | true
+**start\_timestamp**		| string        | Start Timestamp in EEN format: [YYYYMMDDHHMMSS.NNN](#een-timestamp) | ✓
```
Comment (jtallieu) [inline source/includes/_api_pngspan.md]: The properties should be alphabetical

File: source/includes/_api_pngspan.md [hunk 3]
```diff
@@ -11,26 +11,48 @@ content-location: resource actually rendered. absolute start/end ts and either t
 
 ### Discussion
 
-The pngspan is a very efficient mechanism for visualizing where metrics and spans are active. Scale the image vertically as needed. PNG are extremely compact - a day of spans will be a few hundred bytes.
+The pngspan is a very efficient mechanism for visualizing where metrics and spans are active. PNG are extremely compact - a day of spans will be a few hundred bytes. The user is expected to scale the image vertically as needed.
```
Comment (thartley) [inline source/includes/_api_pngspan.md]: Per my note on the other PR for websocket poll, we might need to scale back the verbiage here to be more concise, which is what Dean told me to do with the camera settings section.  I don't think he wants this level of detail.

File: source/includes/_api_pngspan.md [hunk 4]
```diff
@@ -11,26 +11,48 @@ content-location: resource actually rendered. absolute start/end ts and either t
...
 
 Hit detection (for rollover) can be done in a browser by rendering opaque colors and reading pixels values from a one pixel high offscreen image. If an active pixel is detected, fetch the window of events around the timestamp estimate (since the pixel resolution is usually much less than the ms resolution needed for a timestamp), and use the response to determine what metric/span to display (ie the closest one…).
 
 ### PNG Types
 
-  * setting
-  	* table=onoff - "camera_on" setting, the inverse of which represents camera "off"
+<!--- TODO: Convert this int a table? -->
+
+<!--- TODO: Document event type, document purge_types, video_types -->
```
Comment (thartley) [inline source/includes/_api_pngspan.md]: another TODO, is update to reflect Jeff's latest changes to this API

**Comment by ezequielgarcia** [comment]
@jtallieu @thartley @mrauer @bmaassenee This PR is ancient-history-old so I'm closing it. Thanks!

**Pull request closed without merging**